cloudman/values.yaml

```diff
@@ -30,6 +30,10 @@ helmsman_config:
                  - toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.72
                container:
                  docker_container_id_override: quay.io/biocontainers/fastqc:0.11.8--1
+             - tool_ids:
+                 - Grouping1
```

Comment:
@ic4f Should the first letter be lowercase?

Reply:
thats' the id in the tool's xml - Grouping1 (uppercase G)